share/completions/git.fish

```diff
@@ -1435,10 +1445,15 @@ complete -f -c git -n '__fish_git_using_command rebase' -l no-ff -d 'No fast-for
 # This actually takes script for $SHELL, but completing that is... complicated.
 complete -r -c git -n '__fish_git_using_command rebase' -l exec -d 'Execute shellscript'
 
+### reflog
+complete -f -c git -n '__fish_git_needs_command' -a reflog -d 'Manage reflog information'
+complete -f -c git -n '__fish_git_using_command reflog' -a '(__fish_git_branches)'
+complete -f -c git -n '__fish_git_using_command reflog' -a '(__fish_git_heads)' -d 'Head'
+
 ### reset
 complete -c git -n '__fish_git_needs_command' -a reset -d 'Reset current HEAD to the specified state'
 complete -f -c git -n '__fish_git_using_command reset' -l hard -d 'Reset files in working directory'
-complete -c git -n '__fish_git_using_command reset' -a '(__fish_git_branches)'
+complete -c git -n '__fish_git_using_command reset; and not contains -- -- (commandline -op)' -a '(__fish_git_branches)'
```

Comment:
I think we can use `commandline -c` (`--cut-at-cursor`) to keep completing refs if the cursor is left of the `--`.
You can also add the flag `-c` to all other occurrences of `not contains -- -- (commandline -op)` in this file.

Reply:
Neat! Thanks for the tip.